src/static_canvas.class.js

```diff
@@ -119,7 +119,8 @@
     controlsAboveOverlay: false,
 
     /**
-     * Indicates whether the browser can be scrolled when using a touchscreen and dragging on the canvas
+     * Indicates whether the browser can be scrolled when using a touchscreen and dragging on the canvas.
+     * This must a property passed to the canvas when instantiated. Setting this property after, has no effect.
```

Comment:
Can we change this in:
`In order to work, this property has to be passed in the canvas options at creation time. Changing it later won't have any effect`

Reply:
sounds great.